ansible/ipi-ocp-vmware-play/README.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
```

Comment:
This readme might need some words, seems rather empty right now

Reply:
readme updated with details.